routers/repo/milestone.go

```diff
@@ -199,8 +199,9 @@ func EditMilestonePost(ctx *context.Context, form auth.CreateMilestoneForm) {
 	ctx.Redirect(ctx.Repo.RepoLink + "/milestones")
 }
 
-// ChangeMilestonStatus response for change a milestone's status
-func ChangeMilestonStatus(ctx *context.Context) {
+// ChangeMilestoneStatus response for change a milestone's status
+// nolint: dupl
```

Comment:
Switching off a linter rule without an explicit reason for doing so is very bad form. This needs a very explicit comment as to why this is necessary instead of refactoring